### PR TITLE
Porting fs cephx test from nautilus to pacific

### PR DESCRIPTION
**Pull request by AmarnatReddy**
Porting fs cephx test from nautilus to pacific

Tier- 0 : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-ljb4a
Tier -1 : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-rgetu/

Client Suite : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1yul1/
Signed-off-by: Amarnath K <amk@amk.remote.csb>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/cephfs_utilsV1.py [hunk 1]
```diff
@@ -167,17 +167,83 @@ def auth_list(self, clients, **kwargs):
...
-                sudo=True,
-                cmd=f"ceph auth get-or-create client.{client.node.hostname}"
-                f" mon 'allow *' mds "
-                f"'allow *, allow * path=/' osd 'allow *'"
-                f" -o /etc/ceph/ceph.client.{client.node.hostname}.keyring",
-            )
-            client.exec_command(
-                sudo=True,
-                cmd=f"chmod 644 /etc/ceph/ceph.client.{client.node.hostname}.keyring",
-            )
```
Comment (hkadam134) [inline tests/cephfs/cephfs_utilsV1.py:180]: Any other suites this code is being used?

Can we run all other suites also once?

File: tests/cephfs/clients/multiclient_cephx_restrict_fs.py [hunk 2]
```diff
@@ -0,0 +1,306 @@
+import timeit
+import traceback
+
+from ceph.ceph import CommandFailed
+from tests.cephfs.cephfs_utils import FsUtils
+from tests.cephfs.cephfs_utilsV1 import FsUtils as FsUtilsV1
+from utility.log import Log
+
+log = Log(__name__)
+
```
Comment (hkadam134) [inline tests/cephfs/clients/multiclient_cephx_restrict_fs.py:10]: Add Doc String for this TC

Comment (AmarnatReddy) [reply tests/cephfs/clients/multiclient_cephx_restrict_fs.py:10]: Added Doc String

**Review by hkadam134 — APPROVED** [review]
LGTM

**Pull request merged**